src/components/user-list-item.component.js

```diff
@@ -131,7 +133,7 @@ class UserListItemComponent extends Component {
               <Image
                 style={styles.avatar}
                 source={{
-                  uri: user.avatar_url,
+                  uri: imageUrl || user.avatar_url,
```

Comment:
I think it's best to make the `imageUrl` prop mandatory and pass it as { user.avatar_url } where needed

Reply:
I do not think that it will turn out, i.e. when querying GraphQL, another notation is used (lowerCase instead of snake_case). We have already done this, [here](https://github.com/gitpoint/git-point/blob/fe77ed05a87f6221c468977351fa0336ad780c43/src/components/user-list-item.component.js#L143)

Reply:
I still think that having an optional prop, magically provided by another prop in some cases, is a bit confusing. I'm in favor of "real dumb" presentation components.

Also in the spirit of #350 (did you check it?), I think GraphQL responses should be stored using the same schemas as REST responses in the entities array. That would enable us to access the data in the same way, regardless of where it came from (rest or graphql). Am I making sense? :/

Reply:
I have not watched, but probably this will solve the problem, it seems to me...